ceno_zkvm/src/instructions/riscv/dummy/dummy_ecall.rs

```diff
@@ -208,8 +210,14 @@ impl<E: ExtensionField> GKRIOPInstruction<E> for LargeEcallDummy<E, KeccakSpec>
                     .unwrap()
             })
             .collect_vec();
+        let num_instances = instances.len();
 
-        layout.phase1_witness_group(KeccakTrace { instances })
+        layout.phase1_witness_group(KeccakTrace {
```

Comment:
value here is just mock. Will use correct value during integration in another task